include/boost/geometry/algorithms/detail/relate/boundary_checker.hpp

```diff
@@ -85,6 +81,67 @@ class boundary_checker<Geometry, Strategy, linestring_tag>
     Strategy const& m_strategy;
 };
 
+
+template <typename Point, typename Strategy, typename Out>
+inline void copy_pts_boundary_points(Point const& front_pt, Point const& back_pt,
```

Comment:
> What is `pts`?

I later saw you use `seg` and `ls` etc, so that part is consistent. But `pts` followed by `points` looks a bit weird.
Anyway, you can ignore this comment if you want, it's a detail.

Reply:
I agree that it looks wierd. I'll think about a replacement.

Reply:
I renamed this to `copy_boundary_points` and the rest to e.g. `copy_boundary_points_of_mls`.